webrecorder/wabac.js
# adblock support: support for optionally injecting adblock css rules for domain given

Supports filtering of Adblock Plus css hiding rules and serving css to hide the specified selectors.
- Can load and parse a text file containing `<domain>##<css selector>` rules, a subset of adblock plus formatted filters.
- Query param `adblockUrl` specifies the text file
- Can load a gzipped file.
- The endpoint `/mp_/adblock:<domain>` will return css rules for that domain, `<domain>##<css selector>
- The endpoint `/mp_/adblock:` returns all-domain rules, eg. `##<css selector>`
- If adblocking enabled, link to css add to the wombat header insert.
- Fixes #166
- Testing: Update examples/live-proxy to support adblock via index-adblock.html
- bump version to 2.18.0

Testing:
- Run `yarn run build`, then `yarn run start-live-proxy`. This starts a web server on 10001
- Load a site with ads via live proxy, eg: `http://localhost:10001/#https://www.cnn.com/`, confirm ads are shown.
- Load a site with ads through adblock endpoint, eg: `http://localhost:10001/index-adblock.html#https://www.cnn.com/`, confirm adds are not shown.
